pages/configs/api-extractor_json.md

```diff
@@ -339,6 +356,23 @@ The output path for the doc model file.  The file extension should be `.api.json
 The path is resolved relative to the folder of the config file that contains the setting; to change this,
 prepend a folder token such as `<projectFolder>`.
 
+### docModel.includeForgottenExports
+
+Example:
+```js
+  "docModel": {
+    . . .
+    "includeForgottenExports": true,
+    . . .
+  }
+```
+
+**Default value:** `false`
+
+Whether "forgotten exports" should be included in the doc model file. Forgotten exports are declarations
+flagged with `ae-forgotten-export` warnings. See https://api-extractor.com/pages/messages/ae-forgotten-export/
```

Comment:
@zelliot thanks for updating the docs! 

FYI I replaced this URL with a hyperlink